### PR TITLE
added allowSerializedAccess, maxWaitTime and accessOnTimeout to metatype.properties

### PR DESCRIPTION
**Pull request by randyclee**
added allowSerializedAccess, maxWaitTime and accessOnTimeout to metatype.properties

### CODE REVIEW

**Comment by randyclee** [comment]
Fixes #13624
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -136,4 +136,13 @@ extraProperties.name=Additional properties
...
 modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
-modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
\ No newline at end of file
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
+
+allow.serialized.access.name=Allow serialized access
+allow.serialized.access.description=Enable this option to allow serializing access to session data.
+
+max.wait.time.name=Max wait time
+max.wait.time.description=The amount of time, in milliseconds, a servlet waits on a session before continuing execution.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties:145]: Change "a session before continuing execution" to "a session before it continues execution".

**Comment by randog** [comment]
#build

**Comment by randyclee** [comment]
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -136,4 +136,13 @@ extraProperties.name=Additional properties
...
+
+access.on.timeout.name=Access on timeout
+access.on.timeout.description=This property gives the servlet access to the session, allowing normal execution even if the session is still locked by another servlet. Default is true, set to false makes the servlet execution abort when the session requests a time out.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties:148]: Change "allowing normal execution" to "which allows normal execution".

**Comment by randyclee** [comment]
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -136,4 +136,13 @@ extraProperties.name=Additional properties
 extraProperties.description=Additional properties related to session management.
 
 modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
-modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
\ No newline at end of file
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
+
+allow.serialized.access.name=Allow serialized access
+allow.serialized.access.description=Enable this option to allow serializing access to session data.
```
Comment (dmuelle) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties:142]: Enable this option to allow serializing access to session data.

Why is it "serializing" in the description but "serialized" in the name? What's the distinction? If there is none, they should both be "serialized"

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 4]
```diff
@@ -136,4 +136,13 @@ extraProperties.name=Additional properties
...
+
+max.wait.time.name=Max wait time
+max.wait.time.description=The amount of time, in milliseconds, a servlet waits on a session before it continues execution.
+
+access.on.timeout.name=Access on timeout
+access.on.timeout.description=This property gives the servlet access to the session, which allows normal execution even if the session is still locked by another servlet. Default is true, set to false to make the servlet execution abort when the session requests a time out.
```
Comment (dmuelle) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties:148]: `...Default is true, set to false to make the servlet execution abort when the session requests a time out.`

suggested edit:

`Set this property to false to stop the servlet execution when the session requests a timeout. The default value is true.`

**Comment by skasund** [comment]
L2 message review not required

**Comment by rl123456** [comment]
#build

**Comment by randyclee** [comment]
#build

**Comment by randyclee** [comment]
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (zhang-ca) [reply dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties]: In addition, change "in milliseconds" to "in seconds". 
Here's the code : 
`long syncSessionTimeOut = (long)_smc.getSerializedSessionAccessMaxWaitTime() * 1000L; //convert to millisecond`

**Comment by zhang-ca** [comment]
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 5]
```diff
@@ -136,4 +136,14 @@ extraProperties.name=Additional properties
...
 modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
-modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
\ No newline at end of file
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
+
+allow.serialized.access.name=Allow serialized access
+allow.serialized.access.description=Enable this option to allow serialized access to session data.
+
+max.wait.time.name=Max wait time
+max.wait.time.description=The amount of time, in milliseconds, a servlet waits on a session before it continues execution.
```
Comment (zhang-ca) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties]: Change "in milliseconds" to "in seconds" in max.wait.time.description.
The code will convert to millisecond:
long syncSessionTimeOut = (long)_smc.getSerializedSessionAccessMaxWaitTime() * 1000L; //convert to millisecond

**Comment by randyclee** [comment]
#build

**Review by zhang-ca — APPROVED** [review]
Looks good. No need to change Copyright.

**Comment by randyclee** [comment]
#run-libby-bot

**Pull request merged**